src/scss/components/optional/_order-card.scss

```diff
@@ -155,4 +157,36 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
         padding: 0 spacing(x2);
         @include font-size(base--scaleUp, false);
     }
-}
+
+    .c-orderCard-cuisines {
+        color: $grey--mid;
+        margin-top: 0;
+        text-transform: capitalize;
+        font-size: spacing(x2);
```

Comment:
The value here is `spacing` should be a font size value right?

Reply:
If it should be 12px, can just use `@include font-size(small);`